lib/psulib_traject/macros.rb

```diff
@@ -8,6 +8,22 @@
 
 module PsulibTraject
   module Macros
+    # Extracts facets without owner field
+    #
+    # Checks to see if owner is in e subfield, if so it skips
+    def extract_marc_without_owner(fields)
+      lambda do |record, accumulator|
+        return nil unless record.is_a? MARC::Record
+
+        Traject::MarcExtractor.cached(fields).collect_matching_lines(record) do |field, spec, extractor|
+          next if field.subfields.any? { |subfield| subfield.code == 'e' && subfield.value.include?('owner') }
+
+          values = extractor.collect_subfields(field, spec)
+          accumulator.concat(values) if values && !values.empty?
```

Comment:
could this be consolidated to `if values.present?`

Reply:
I changed it to values.present? but now it errors out. If it ain't broke...